pkgs/by-name/ne/nerdfetch/package.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitea
+}:
+stdenv.mkDerivation rec {
```

We are currently in the procces of switching from rec to using finalAttrs

```suggestion
stdenv.mkDerivation (finalAttrs: {
```